src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common.Tooling/LanguageServer/LanguageServerTestBase.cs

```diff
@@ -131,11 +131,36 @@ private protected static RazorLSPOptionsMonitor GetOptionsMonitor(bool enableFor
     {
         var configService = StrictMock.Of<IConfigurationSyncService>();
 
-        var options = new RazorLSPOptions(enableFormatting, true, InsertSpaces: true, TabSize: 4, autoShowCompletion, autoListParams, formatOnType, autoInsertAttributeQuotes, colorBackground, codeBlockBraceOnNextLine, commitElementsWithSpace);
+        var options = new RazorLSPOptions(
+            GetFormattingFlags(enableFormatting, formatOnType),
+            true,
+            InsertSpaces: true,
+            TabSize: 4,
+            autoShowCompletion,
+            autoListParams,
+            autoInsertAttributeQuotes,
+            colorBackground,
+            codeBlockBraceOnNextLine,
+            commitElementsWithSpace);
         var optionsMonitor = new RazorLSPOptionsMonitor(configService, options);
         return optionsMonitor;
     }
 
+    private static FormattingFlags GetFormattingFlags(bool enableFormatting, bool formatOnType)
+    {
+        if (!enableFormatting)
+        {
+            return FormattingFlags.Disabled;
+        }
+
+        if (formatOnType)
+        {
+            return FormattingFlags.All;
```

Comment:
Why isn't this `FormattingFlags.OnType` ?

Reply:
I guess it would be `FormattingFlags.Enabled | FormattingFlags.OnType` but I didn't add something for paste